gateway/src/node/client_handling/websocket/shared_state.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright 2024 - Nym Technologies SA <contact@nymtech.net>
+// SPDX-License-Identifier: GPL-3.0-only
+
+use crate::node::client_handling::websocket::connection_handler::coconut::CoconutVerifier;
+use crate::node::client_handling::websocket::connection_handler::BandwidthFlushingBehaviourConfig;
+use nym_crypto::asymmetric::identity;
+use std::sync::Arc;
+
+// I can see this being possible expanded with say storage or client store
+#[derive(Clone)]
+pub(crate) struct SharedHandlerState {
```

Comment:
With something called `Shared` I'd imagine the user expectation is that it's all behind an Arc? That is, if the boolean there is updated it's reflected in other shared instances as well?